### DISCUSSION TITLE
<CMake Error at CMakeLists.txt:6 (find_package)>

### DISCUSSION THREAD

**Discussion by ghost** [community/community#40919] [Actions]
### Select Topic Area

Question

### Body

When I "cmake ." and enter, this error keeps come out. As you can see in this picture, I already included OpenCVConfig.cmake's path, but the error doesn't disappear.

Any ideas..?

![CMakeLists txt에 OpenCVConfig cmake 경로 추가한 상태](https://user-images.githubusercontent.com/119941744/205911091-59a7ea38-974c-426d-90e2-7f388e16d014.PNG)
![cmake   입력한 결과](https://user-images.githubusercontent.com/119941744/205911111-f670b922-1e52-43b8-af01-4143590fa92c.PNG)
![OpenCVConfig cmake 위치](https://user-images.githubusercontent.com/119941744/205911121-7c0eaf59-ab74-47c6-8719-1649aa7e9644.PNG)

**Comment by ossanna16** [comment]
Hi there @vampy0914 and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
